src/pages/index.js

```diff
@@ -122,5 +241,7 @@ initialCards.forEach((cardData) => renderCard(cardData));
 
```

Comment:
Please, delete `this.disableButton(this._submitButton,this._config.inactiveButtonClass);` from line 71 from `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.